src/chat.rs

```diff
@@ -24,6 +24,22 @@ use crate::param::*;
 use crate::sql;
 use crate::stock::StockMessage;
 
+/// An chat item, such as a message or a marker.
+#[derive(Debug, Copy, Clone)]
+pub enum ChatItem {
+    Message {
+        msg_id: MsgId,
+    },
```

Comment:
I'm curious why you chose a struct with named fields rather then a tuplestruct.  Not that this is wrong, I don't mind.  Only interested in the reasoning.

Reply:
Message *has* id, while `MsgId` *is* the u32 which it wraps. I am pretty sure we are not going to add anything to `MsgId` "structure", but might want to add timestamp or something else to the chat item.